src/components/VideoPlayer2.tsx

```diff
@@ -54,6 +54,16 @@ export const VideoPlayer: FunctionComponent<VideoPlayerProps> = ({
     if (!player) {
       return;
     }
+    const activeElement = document.activeElement;
+
+        // Check if there is an active element and if it's an input or textarea
+    if (
```

Comment:
This wont work
activeElemeent changing doens't re-trigger the effect
So you can have a stale element here

Reply:
Thanks for the review @hkirat , got the mistake I made, idk how that worked though!